packages/babel-plugin-transform-inline-consecutive-adds/src/index.js

```diff
@@ -0,0 +1,193 @@
+"use strict";
+
+const COLLAPSERS = [
+  "./object-collapser",
+  "./array-collapser",
+  "./array-property-collapser",
+  "./set-collapser"
+].map((x) => {
+  const Collapser = require(x);
```

Comment:
use `require("./object-collapser")`. Do not use dynamic requires - for bundling purposes.